CHANGELOG.md

```diff
@@ -3,6 +3,25 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.2 - Gray Eagle [11/01/2023]
+
+### `Added`
+
+- Update module pbccs (v6.4.0)
+- Update module isoseq refine (v3.8.1)
+- Update module lima (v2.6.0)
+- Update module uLTRA (v0.0.4.2). Fix [issue #17](https://github.com/ksahlin/ultra/issues/17). Thanks to [Husen M. Umer](https://github.com/husensofteng).
+- Zenodo DOI
+- Update to template v2.7.2
```

Comment:
In many pipelines the PR that made the change is listed here, just as future suggestion.